examples/azure-native-sample-project/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Native sample project
+
+This project creates a VM instance in Azure Cloud using the Pulumi Kotlin Azure native SDK.
+
+## How to run
+
+1. Install Pulumi (see: [docs](https://www.pulumi.com/docs/install/)).
+2. Log in to your Pulumi account (see: [docs](https://www.pulumi.com/docs/cli/commands/pulumi_login/)).
+   ```bash
+   pulumi login
+   ```
+3. Install the Azure CLI (see: [docs](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)).
```

Comment:
I know that this will be different for each OS, but maybe we should copy & paste the (potentially) most popular one-liner (macOS)?

`brew install azure-cli`